### PR TITLE
LPS-136688 Save and retrieve numeric data correctly

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-136688

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java [hunk 1]
```diff
@@ -272,7 +285,10 @@ private String _getObjectFieldName(DDMFormField ddmFormField) {
 		}
 	}
 
-	private Object _getValue(String objectFieldType, String value) {
+	private Object _getValue(
+			String objectFieldType, String value, Locale defaultLocale)
```
[the remark this answers is not in the document]
Comment (mateussgn) [reply modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java]: Sure!

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:reevaluate:1277401_803

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:forward

**Pull request closed without merging**